backend/src/test/java/com/woowacourse/levellog/presentation/OAuthControllerTest.java

```diff
@@ -15,22 +19,51 @@
 @DisplayName("OAuthController의")
 class OAuthControllerTest extends ControllerTest {
 
-    @ParameterizedTest
-    @ValueSource(strings = {" "})
-    @NullAndEmptySource
-    @DisplayName("login 메서드는 코드에 공백이나 null이 들어오면 예외를 던진다.")
-    void login(final String code) throws Exception {
-        // given
-        final GithubCodeRequest request = new GithubCodeRequest(code);
-        final String requestContent = objectMapper.writeValueAsString(request);
-
-        // when
-        final ResultActions perform = mockMvc.perform(post("/api/auth/login")
-                        .contentType(MediaType.APPLICATION_JSON)
-                        .content(requestContent))
-                .andDo(print());
-
-        // then
-        perform.andExpect(status().isBadRequest());
+    private static final String AUTHORIZATION_CODE = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiI0IiwiaWF0IjoxNjU4ODk3MjQ0LCJleHAiOjE2NTg5MzMyNDR9";
+
+    @Nested
+    @DisplayName("login 메서드는")
+    class login {
+
+        @ParameterizedTest
+        @ValueSource(strings = {" "})
+        @NullAndEmptySource
+        @DisplayName("코드에 공백이나 null이 들어오면 예외를 던진다.")
+        void login_nullAndEmptyCode_exceptionThrown(final String code) throws Exception {
```

Comment:
```suggestion
        void nullAndEmptyCode_exceptionThrown(final String code) throws Exception {
```
는 어떠신가요? 메서드 명에 대한 정보는 클래스에서 충분히 표현될 것 같아요 😁